smart_campus/app/urls.py

```diff
@@ -15,5 +15,6 @@
     url('^update_user_experience_point/$', views.update_user_experience_point, name='Update user experience point'),
     url('^add_user_favorite_stations/$', views.add_user_favorite_stations, name='Add user favorite stations'),
     url('^remove_user_favorite_stations/$', views.remove_user_favorite_stations, name='Remove user favorite stations'),
-    url('^get_all_travel_plans/$', views.get_all_travel_plans, name='Get all travel plans')
+    url('^get_all_travel_plans/$', views.get_all_travel_plans, name='Get all travel plans'),
+    url(r'^add_user_reward/$', views.add_user_reward,name="Add User Reward")
```

Comment:
Add space between `views.add_user_reward` and `,name="Add User Reward"`.
Also, you should follow the naming style of other urls.

Reply:
okay, fixed it.

Reply:
May I ask why there didn't need the pattern "r" ?